src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -349,12 +349,12 @@ bool Thread::CatchAtSafePoint()
 
 uint64_t Thread::GetPalThreadIdForLogging()
 {
-    return *(uint64_t*)&m_threadId;
+    return m_threadId;
 }
 
 bool Thread::IsCurrentThread()
```

Comment:
This method does not have good perf characteristics (PalGetCurrentOSThreadId is a syscall in many cases).

I do not see where it is used. Can it be deleted?